### PR TITLE
Case 21728 - tab in marketplace search bar causes an update of search results

### PR DESCRIPTION
**Pull request by roxanneskelly**
https://highfidelity.manuscript.com/f/cases/21728/tab-in-marketplace-search-bar-causes-an-update-of-search-results

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18151/HighFidelity-Beta-PR15191-b220b76.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18151/HighFidelity-Quest-Beta-PR15191-b220b76.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18151/HighFidelity-Beta-PR15191-b220b76.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18151/HighFidelity-Beta-Interface-PR15191-b220b76.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18151/HighFidelity-Beta-PR15191-b220b76.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18151/HighFidelity-Beta-Interface-PR15191-b220b76.exe

File: interface/resources/qml/hifi/commerce/marketplace/Marketplace.qml [hunk 1]
```diff
@@ -359,9 +359,11 @@ Rectangle {
                 }
 
                 onAccepted: {
-                    root.searchString = searchField.text;
-                    getMarketplaceItems();
-                    searchField.forceActiveFocus();
+                    if(root.searchString !== searchField.text) {
```
Comment (howard-stearns) [inline interface/resources/qml/hifi/commerce/marketplace/Marketplace.qml]: coding standard: whitespace between if and paren

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18265/HighFidelity-Beta-PR15191-27fa0dc.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18265/HighFidelity-Beta-Interface-PR15191-27fa0dc.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18265/HighFidelity-Beta-PR15191-27fa0dc.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18265/HighFidelity-Quest-Beta-PR15191-27fa0dc.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18265/HighFidelity-Beta-PR15191-27fa0dc.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18265/HighFidelity-Beta-Interface-PR15191-27fa0dc.exe

File: interface/resources/qml/hifi/commerce/marketplace/Marketplace.qml (hunk 1 above)
Comment (roxanneskelly) [reply interface/resources/qml/hifi/commerce/marketplace/Marketplace.qml]: fixed

**Pull request merged**